HelloMixpanel/HelloMixpanelTests/MiscEdgeCaseTests.m

```diff
@@ -0,0 +1,28 @@
+//
+//  MiscEdgeCaseTests.m
+//  HelloMixpanel
+//
+//  Created by Peter Chien on 8/11/17.
+//  Copyright © 2017 Mixpanel. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "MixpanelBaseTests.h"
+
+@interface MiscEdgeCaseTests : MixpanelBaseTests
+
+@end
+
+@implementation MiscEdgeCaseTests
+
+- (void)testInitializeMixpanelOnBackgroundThread {
```

Comment:
does this really need a whole other file, why not put it part of the base tests?

Reply:
it didn't feel like a base test but didn't fit into any of our current files. was more like a one off

Reply:
actually now that i'm looking MixpanelBaseTests doesn't actually have any tests at all. not sure if putting tests in there would run in all the subclasses

Reply:
Here should be fine: https://github.com/mixpanel/mixpanel-iphone/blob/master/HelloMixpanel/HelloMixpanelTests/HelloMixpanelTests.m

Reply:
i think this doesn't deserve a separate file especially when it's called MiscEdgeCaseTests

Reply:
ah i'll move it there. didn't realize that was already a mix of different tests